packages/headless/src/features/index.ts

```diff
@@ -28,6 +28,8 @@ export * from './question-answering/question-answering-actions-loader';
 export * from './breadcrumb/breadcrumb-actions-loader';
 export * from './recent-queries/recent-queries-actions-loader';
 export * from './recent-results/recent-results-actions-loader';
+export * from './case-fields/case-fields-actions-loader';
+export * from './case-inputs/case-inputs-actions-loader';
```

Comment:
I believe these exports will be exported in the search use-case.

Suggestion: personally, I'd place all exports directly in the `case-assist.indext.ts` file.